tests/integration/test_xnat2bids.py

```diff
@@ -0,0 +1,93 @@
+from xnat_tools import __version__
+import subprocess
+from subprocess import Popen, PIPE, STDOUT
+import os
+import glob
+import shutil
+import shlex
+import json
+from dotenv import load_dotenv
+load_dotenv()
+
+def test_xnat2bids():
+    """Integration test for xnat2bids executable"""
+
+    xnat_user = os.environ.get("XNAT_USER", "testuser")
+    xnat_pass = os.environ.get("XNAT_PASS", "")
+    session = os.environ.get("XNAT_SESSION", "")
+    session_suffix = os.environ.get("XNAT_SESSION_SUFFIX", "01")
+    bids_root_dir = os.environ.get("XNAT_BIDS_ROOT", "./tests/xnat2bids")
+    seqlist = ['1', '2', '3', '6']
+    skiplist = ['2', '3']
+
+    if os.path.exists(bids_root_dir):
+        shutil.rmtree(bids_root_dir, ignore_errors=True)
+
+    os.mkdir(bids_root_dir)
+    
+   
+    xnat2bids_cmd = f"xnat2bids --user {xnat_user} --password {xnat_pass} \
+                      --session {session} --session_suffix {session_suffix} \
+                      --bids_root_dir {bids_root_dir} \
+                      --seqlist {' '.join(seqlist)} --skiplist {' '.join(skiplist)} \
+                      -vv"
+
+
+    xnat2bids_split_cmd = shlex.split(xnat2bids_cmd)
+    
+    subprocess.run(xnat2bids_split_cmd)
+
+    filepath = f"tests/xnat2bids/*/study-*/bids/sub-*/ses-{session_suffix}"
+
+    #***************************************************************************
+    # Test for succesfull execution
+    #***************************************************************************
+    assert len(glob.glob(f"{filepath}/*/*.json")) > 0
+    assert len(glob.glob(f"{filepath}/*/*.nii.gz")) > 0
```

Comment:
Do we know how many files we should have? Do you think this assertion captures success accurately?

Reply:
Because we can specify any session, no we don't know how many files we have. So in that sense, success means it run through and it created some `json` and `nii.gz` files. I think we can make the tests more sequence specific when we create a testing dataset